### PR TITLE
Simplify linking content

### PR DESCRIPTION
**Pull request by imatwawana**
This is an initial **VERY ROUGH** draft of an envisioned restructuring of the [Links and cross references](https://grafana.com/docs/writers-toolkit/write/references/) page. The purpose of this page is to tell the writer: 

1. What kind of link should I use?
2. How do I format it correctly?

Right now the page requires you to read—rather than scan—to figure out the first question and requires a lot of reading to understand the second part when it comes to `relref` shortcodes specifically. This PR attempts to simplify the page so that you can quickly tell which part of the content to focus on and then quickly understand what you need to do. This PR also attempts to remove any information that's not essential to answering questions 1 and 2 and attempts to remove any jargon that isn't easily understood from context/explained or doesn't help with the task.

**NOTE FOR REVIEWERS**: Purpose of review _**at this stage**_ is to confirm the efficacy of this format and to check the accuracy.

### CODE REVIEW

File: docs/sources/write/references/index.md [hunk 1]
```diff
@@ -15,109 +15,91 @@ keywords:
...
-   Hugo emits logs during the build for broken links defined with the `relref` shortcode.
-   For more information about the `relref` shortcode, refer to [Build time link checking with Hugo](#build-time-link-checking-with-hugo).
+- the `relref` shortcode
+- partial URL with relative path
 
-1. Otherwise, use the fully qualified URL.
+For example: `{{</* relref "./path/to/page" */>}}`.
 
-   For example, `[GitHub](https://github.com)`, or `[Grafana](https://grafana.com/docs/grafana/latest/)`.
+### Buildtime link checking and Hugo error output
```
[the remark this answers is not in the document]
Comment (imatwawana) [reply docs/sources/write/references/index.md:47]:
```suggestion
### Build time link checking and Hugo error output
```

File: docs/sources/write/references/index.md [hunk 2]
```diff
@@ -15,109 +15,91 @@ keywords:
...
 
-#### Example
+In this case, the source content is in the 1999 F150 folder and the destination content is in the Vans folder.
+The common folder for the two pieces of content is the Vehicles folder.
+The number of folders between the source folder, 1999 F150 and the common folder Vehicles is 3 folders, so this requires three aliases: ../../../
+The pathway from the common Vehicles to destination folder Vans /vans
+The relative path is ../../../vans
 
-In the Writers' Toolkit repository, with the following directory structure:
+If the source folder was Vans and the desitination was 1999 F150, the pathway would be ../trucks/F150/1999-F150
```
[the remark this answers is not in the document]
Comment (imatwawana) [reply docs/sources/write/references/index.md:89]:
```suggestion
If the source folder was Vans and the destination was 1999 F150, the pathway would be ../trucks/F150/1999-F150
```

**Comment by jdbaldry** [comment]
I think the broad structure is good and I appreciate the demonstration of it.

I will be revising these docs for technical accuracy and updated guidelines today so will take this structure and work from there.

**Comment by imatwawana** [comment]
Replaced by #345

**Pull request closed without merging**